nttt/acknowledgements.py

```diff
@@ -0,0 +1,74 @@
+import math
+import pandas
+import sys
+from .utilities import get_file, save_file
+
+
+def get_volunteer_acknowledgement(csv_file_path, language):
+    '''
+    Returns the volunteer acknowledgement for the given language or None if
+    the volunteer acknowledgement doesn't exist.
+    '''
+
+    data_frame = None
+    try:
+        data_frame = pandas.read_csv(csv_file_path)
+    except:
+        print("Could not read the volunteer acknowledgements spreadsheet.",
+              file=sys.stderr)
+        return None
+
+    acknowledgement = None
+    for i in data_frame.index:
+        if data_frame.at[i, 'Locale'] == language:
+            acknowledgement = data_frame.at[i, 'Acknowledgement']
+            break
+    return acknowledgement
+
+
+def add_volunteer_acknowledgement(csv_file_path, output_file_path, language,
+                                  volunteers, logging):
+    '''
+    Adds the volunteer acknowledgement to the given output file. Returns
+    whether the volunteer acknowledgement was successfully added.
+    '''
+
+    acknowledgement = get_volunteer_acknowledgement(csv_file_path, language)
+    if acknowledgement is None:
+        print("Could not find the volunteer acknowledgement for language {}".format(language),
+              file=sys.stderr)
+        return False
+    elif str(acknowledgement) == "nan":
```

Comment:
In case of an empty cell, pandas does not return an empty string (or None), but a float value NaN. In string format, this becomes 'nan'. Added a constant with some comment.